.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: 🐛 Bug Report
+about: Bugs, missing documentation, or unexpected behavior 🤔.
+---
+
+<!--
+
+* Please fill out this template with all the relevant information so we can
+  understand what's going on and fix the issue. We appreciate bugs filed and PRs
+  submitted!
+
+-->
+
+- `@freenow/wave` version:
+
+### Relevant code
+
+```js
+var your => (code) => here;
+```
+
+### What you did
+
+<!-- What you were doing -->
+
+### What happened
+
+<!-- Please provide the full error message/screenshots/anything -->
+
+### Reproduction
+
+<!--
+If you can, try to reproduce the issue in a Codesandbox. You can fork the one
+here: https://codesandbox.io/s/wave-playground-w7cf3 (make sure to use latest Wave version)
+-->
+
+### Problem description
+
+<!-- Please describe why the current behavior is a problem -->
+
+### Suggested solution
```

Comment:
Not sure if I would include this section in the bug report already. It discourages ideas from the community in the comments of a bug